src/data/activities.json

```diff
@@ -62,5 +62,181 @@
     "personID": "default",
     "created": "old",
     "displayHeader": true
+  },
+  "hellen-keller-1": {
```

Comment:
great, more data (y)

Reply:
Totally added it by accident, and will remove it